content/02.intro.md

```diff
@@ -2,11 +2,11 @@
 
 Deep learning (DL), a subfield of machine learning (ML) implementing artificial neural networks with many layers, is increasingly used for the analysis of biological data [@doi:10.1098/rsif.2017.0387].
 In many cases, novel biological insights have been revealed through careful evaluation of DL methods ranging from predicting protein-drug binding kinetics [@doi:10.1038/s41467-017-02388-1] to identifying the lab-of-origin of synthetic DNA [@doi:10.1038/s41467-018-05378-z].
-However, a lack of relevant up-to-date resources focused on biological applications of DL pose a significant barrier to newcomers wishing to utilize state-of-the-art DL in their research.
+However, a lack of concise recommendations for biological applications of DL hamper newcomers wishing to apply state-of-the-art DL in their research.
 As DL is an active and specialized research area, detailed resources are rapidly rendered obsolete and few resources articulate general DL best practices to the scientific community.
-Most instructional literature focuses on ML broadly, rather than DL specifically, further limiting accessibility and reproducibility.
-To address this issue, we solicited input from a diverse community of researchers, who wrote this manuscript collaboratively using the GitHub version control platform [@url:https://github.com/Benjamin-Lee/deep-rules] and Manubot [@url:https://greenelab.github.io/meta-review/].
```

Comment:
We don't appear to be that diverse from author names, so I don't think we should say we are.

Reply:
I realize that I may not have been entirely clear in this line. What about: "we solicited input from a community of researchers with varied biological and deep learning interests"